### PR TITLE
Text and style fixes for community pages

### PR DESCRIPTION
**Pull request by svrnm**
Some minor fixes to the end user pages.

### CODE REVIEW

File: content/en/community/_index.md [hunk 1]
```diff
@@ -16,6 +16,6 @@ our [community repo](https://github.com/open-telemetry/community).
 
 ## Open Source Collaboration
 
-OpenTelemetry collaborates and integrates with other open-source projects
+OpenTelemetry collaborates and integrates with other open source projects
```
Comment (chalin) [inline content/en/community/_index.md:19]: In this case I'd say that the original is correct.

**Review by chalin — APPROVED** [review]
See inline comment.

File: content/en/community/_index.md (hunk 1 above)
Comment (svrnm) [reply content/en/community/_index.md:19]: I applied this change because of https://github.com/cncf/foundation/blob/main/style-guide.md

Comment (chalin) [reply content/en/community/_index.md:19]: Ok, that works for me.

**Pull request merged**